### PR TITLE
Provide documentation for v0.3.0

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ PM > Install-Package Arcus.Messaging.* -Version 0.1.0
...
 
 - Support for using message pumps for the following brokers:
-    - Azure Service Bus ([docs](features/message-pumps/service-bus))
-    - Azure Service Bus Extensions ([docs](features/service-bus))
+    - Azure Service Bus ([docs](features/message-pumps/service-bus) | [extensions](features/service-bus))
 - Support for exposing TCP health probes to periodically check liveness/readiness of the host ([docs](features/tcp-health-probe))
 - Customize message pumps ([docs](features/message-pumps/customization))
+    - Fallback message handlers
 
 ## Older versions
```
Comment (stijnmoreels) [inline docs/index.md:27]: Here you can now add the v0.2 as 'older version' since we released v0.3.

**Review by stijnmoreels — APPROVED** [review]

**Comment by stijnmoreels** [comment]
It's fun to see that you found your way arround the whole 'preview -> versioned feature' docs. It's I think a good way to keep track of things. Thanks!

File: docs/index.md (hunk 1 above)
Comment (tomkerkhove) [reply docs/index.md:27]: Thanks for the heads up!

**Comment by stijnmoreels** [comment]
OK, is all good for me. I see that [this](https://deploy-preview-117--arcus-messaging-staging.netlify.app/v0.2.0/) page is still refering to the v0.1 NuGet package; but not blocking here. I'll change that myself if you want to merge this.

File: docs/v0.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Home"
+layout: default
+---
+
+# Installation
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Messaging.Abstractions?packageVersion=0.1.0)](https://www.nuget.org/packages/Arcus.Messaging.Abstractions/0.3.0)
```
Comment (stijnmoreels) [inline docs/v0.3.0/index.md:8]:
```suggestion
[![NuGet Badge](https://buildstats.info/nuget/Arcus.Messaging.Abstractions?packageVersion=0.3.0)](https://www.nuget.org/packages/Arcus.Messaging.Abstractions/0.3.0)
```

**Review by stijnmoreels — APPROVED** [review]

**Comment by stijnmoreels** [comment]
Thanks for including these fixing of the versions in here 😓 . Some issue we didn't saw previously apparently.

**Pull request merged**